TestTicTacToe/BoardTests.cs

```diff
@@ -0,0 +1,65 @@
+using TicTacToe;
+using Xunit;
+
+namespace TestTicTacToe
+{
+    public class BoardTests
```

Comment:
I think I would like to see a test for GetBoard(). Even though it is kinda covered in the other tests. It could be good to show that after a board is instantiated it returns an empty board (i.e full of zeros).